pkg/generate/app/env.go

```diff
@@ -158,19 +199,21 @@ func LoadEnvironmentFile(filename string, stdin io.Reader) (Environment, error)
 	if info, err := os.Stat(filename); err == nil && info.IsDir() {
 		return nil, fmt.Errorf("Cannot read variables from %q: is a directory", filename)
 	} else if err != nil {
-		return nil, fmt.Errorf("Cannot stat %q: %s", filename, err)
+		return nil, fmt.Errorf("Cannot stat %q: %v", filename, err)
 	}
 
 	env, err := godotenv.Read(filename)
```

Comment:
godotenv.Read() returns a map, which will screw up the k/v ordering when reading in the file.  I don't particularly mind, but perhaps the BZ submitter will...
This is also the only case where godotenv is used within origin.  I'd be inclined to blitz the dependency and TempFile nonsense above and read the file in properly by steam.